pages/common/gh-workflow.md

```diff
@@ -0,0 +1,36 @@
+# gh workflow
+
+> List, view, and run GitHub Actions workflows.
+> More information: <https://cli.github.com/manual/gh_workflow>.
+
+- Interactively select a workflow to view the latest jobs:
+
+`gh workflow view`
+
+- View a specific workflow in the default browser:
+
+`gh workflow view {{id|workflow_name|filename.yml}} --web`
+
+- Display the YAML definition of a specific workflow:
+
+`gh workflow view {{id|workflow_name|filename.yml}} --yaml`
+
+- Display the YAML definition for a specific Git branch or tag:
+
+`gh workflow view {{id|workflow_name|filename.yml}} --ref {{branch__or_tag_name}} --yaml`
+
+- List workflow files (use `--all` to include disabled workflows):
+
+`gh workflow list`
+
+- Run a manual workflow with parameters:
+
+`gh workflow run {{id|workflow_name|filename.yml}} -f {{param1}}={{value1}} -f {{param2}}={{value2}}`
+
+- Run a manual workflow using a specific branch or tag with JSON parameters from stdin:
```

Comment:
Perhaps this will read better?
```suggestion
- Run a workflow manual using a specific branch or tag with JSON parameters from stdin:
```

Reply:
Did you mean `...workflow manually...`, @sbrl?

Reply:
This command is for workflow that have `workflow_dispatch`, so you start them manually:
```yaml
name: Manual Workflow
on: workflow_dispatch
jobs:
```
https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Reply:
Oops! Fixed.

Oh,m I see @navarroaxel. So this only works with workflows that are explicitly defined to be manually triggers? In that case, we should leave this as-is and not apply my suggestion here.

Reply:
Exactly, this only works for workflow with a manual triggers.